### ISSUE TITLE
Strange problem with sound 

### ISSUE THREAD

**Issue by Isakku** [wwmm/pulseeffects#368]
Simmilar to https://github.com/wwmm/pulseeffects/issues/1
I have a dual boot setup. Linux Mint Cinnamon 18.3 and Linux Mint KDE 18.3  Since the hardware is a HP Envy laptop with BeatsAudio, I have HDA jack retask installed and pins are remapped in both setups. 
However
Pulseeffects was installed in both cases from my distro's software manager which for this app is a flatpak, not sure if that could have any influence.
Anyways. Sound works normally in the first case. In the second case (KDE), it doesn't work well. 

Basically I get no sound if I enable any of the effects for any of the running apps (Amarok, VLC, etc.) 
Pulseeffects shows the animation of the sound waves in it's interface, but the actual sound is nowhere to be found. The "Enable all applications" button doesn't seem to do anything.

**Comment by wwmm** [comment]
You are probably using Pulseaudio 11. It has a few problems that only manifest in KDE. The only real solution to this is upgrading to Pulseaudio 12. If that is not possible you will have to unload the module `module-device-manage` before starting PulseEffects. Follow step 3 in the [FAQ](https://github.com/wwmm/pulseeffects/wiki/FAQ)

**Comment by wwmm** [comment]
Only now I saw your edit. Before doing something radical try the points `3,4 and 5` in our [FAQ](https://github.com/wwmm/pulseeffects/wiki/FAQ)

**Comment by wwmm** [comment]
As far as I remember an upgrade to Pulseaudio 12 fixed this problem for all the users that had it

**Comment by mikhailnov** [comment]
> So, maybe if I upgrade to Mint 19

Mint 19 (Ubuntu 18.04) has Pulseaudio 11, you will have to install either pulseaudio 12 only or both pulseeffects and pulseaudio from PPA https://github.com/wwmm/pulseeffects/wiki/Package-Repositories#debian--ubuntu

**Comment by Isakku** [comment]
Since I prefer stability (I already had to mess with HDA jack retask because this laptop comes with beat audio, and I had already installed KDE on top of Cinnamon, it's a pity Mint makers left us KDE users stranded) , I guess I'm screwed until Ubuntu and Mint decide to upgrade to Pulseaudio 12, which I don't think will happen untill the next Ubuntu LTS release, I really doubt Mint 19.1 will do it if they haven't done it for Ubuntu 18.04.
